### ISSUE TITLE
Questions Regarding Completion Length Change in Reproducing SimpleRL-Reason

### ISSUE THREAD

**Issue by nonstopfor** [huggingface/open-r1#465]
I have run the following commands as shown in Readme to reproduce the results in [SimpleRL-Reason](https://hkust-nlp.notion.site/simplerl-reason). 
```
ACCELERATE_LOG_LEVEL=info accelerate launch --config_file recipes/accelerate_configs/zero2.yaml \
    --num_processes=7 src/open_r1/grpo.py \
    --config recipes/Qwen2.5-Math-7B/grpo/config_simple_rl.yaml
```
And my training dynamics are as follows (I slightly modify the config to remove format reward as I observed that it stays at zero during training):

<img width="1258" alt="Image" src="https://github.com/user-attachments/assets/d3ec8f4f-784a-49b0-95fb-fbf23de7b347" /> 

The accuracy indeed increases a lot, which suggests the training process may be correct. However, I note that the completion length keeps decreasing, while it is expected to increase after around 20 training steps, as reported in [SimpleRL-Reason](https://hkust-nlp.notion.site/simplerl-reason).

Can anyone explain about this?

**Comment by TimeLovercc** [comment]
Me too. Format reward keeps 0 all the way. But actually the format reward of this one is different from the format reward of [SimpleRL-Reason](https://hkust-nlp.notion.site/simplerl-reason). Is the difference the reason?

**Comment by linkezh** [comment]
Are you guys using the default setting? Why does my accuracy award keep decreasing 😢

**Comment by TimeLovercc** [comment]
@linkezh My global batch size is smaller, and other settings are same. The accuracy reward keeps increasing.

**Comment by linkezh** [comment]
@TimeLovercc Thanks! I guess I must have screwed something up. 😰

**Comment by 0205090923** [comment]
Exactly the same observation as yours. And how did you eval qwen-math-7b？ its maximum length is 4096, and the given script failed.

**Comment by nonstopfor** [comment]
> Exactly the same observation as yours. And how did you eval qwen-math-7b？ its maximum length is 4096, and the given script failed.

I just run the training part currently

**Comment by james-yw** [comment]
Have you found any reasons? I encountered the same issue when reproducing it using the VERL framework, with Qwen2-7B-Instruct as base model.
